ClioInput - version @@VERSION@@ - build @@BUILD@@ @@DATE@@
10-8-2018 14-3

Processing data file /Users/jrc/mhk-home/sources/soure/documents/casamentos/c1690.cli
-------------------------------------------
Generic Act translation module with geoentities (XML).
     Joaquim Ramos de Carvalho (joaquim@uc.pt) 
** New document: kleio
=========================
kleio translation started
=========================
Structure: gacto.str
Prefix: 
Autorel: 
Translation count: 1
Obs: 
** Processing source fonte$casamentos 1690
6: cas$c1690.1
40: cas$c1690.2

/Users/jrc/mhk-home/sources/soure/documents/casamentos/c1690.cli:55:destination id of "same as" not found. Will check again at the end of file.


WARNING: c1690.cli line 54 destination id of "same as" not found. Will check again at the end of file.
Near line: 54          test$paulo ribeiro cabral/mesmo_que=c24-7/id=c1690.2-per7
Near line: 55             ls$cargo/capitao-mor

71: cas$c1690.3
107: cas$c1690.4
134: cas$c1690.5
165: cas$c1690.6

/Users/jrc/mhk-home/sources/soure/documents/casamentos/c1690.cli:177:destination id of "same as" not found. Will check again at the end of file.


WARNING: c1690.cli line 176 destination id of "same as" not found. Will check again at the end of file.
Near line: 176 
Near line: 177          test$andre goncalves/id=ctest17041690

186: cas$c1690.7
219: cas$c1690.8
252: cas$c1690.9

/Users/jrc/mhk-home/sources/soure/documents/casamentos/c1690.cli:270:destination id of "same as" not found. Will check again at the end of file.


WARNING: c1690.cli line 269 destination id of "same as" not found. Will check again at the end of file.
Near line: 269          test$sebastiao machado da costa/mesmo_que=c12-8/id=c1690.9-per7
Near line: 270             ls$nome/sebastiao machado

279: cas$c1690.10
302: cas$c1690.11
325: cas$c1690.12
344: cas$c1690.13
367: cas$c1690.14
390: cas$c1690.15
412: cas$c1690.16
*** End of File
ERROR:  Could not find same_as id c24-7 referred in line 55. If  original reference is on another file use xsame_as

ERROR:  Could not find same_as id padre_alvaro referred in line 177. If  original reference is on another file use xsame_as

ERROR:  Could not find same_as id c12-8 referred in line 270. If  original reference is on another file use xsame_as

3  errors. 
3  warnings.
Translation finished.
